### ISSUE TITLE
Being told to NOT use my PM voice

### ISSUE THREAD

**Issue by phillipou** [phillipou94/phil-react-blog#8]
![probably how I looked after being told to stop using my “PM voice”](https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F8336812b-b147-47e0-8b9b-9b06c5489fb1_1823x2290.jpeg)

probably how I looked after being told to stop using my “PM voice”

Bookie is supposed to be a much-needed creative outlet from work. Writing about work not only didn’t seem all that fun, but also completely antithetical to what I was trying to accomplish with this blog in the first place.

Maybe it’s the move to New York. Maybe it’s meeting new people and having to answer: “So what do you do?”. Or maybe it's just a good-old-fashioned quarter-life crisis.

Either way, here we are feeling introspective, using our precious free-time to take a stab at writing a tiny career memoir.

I'm not presumptuous enough to force a “What is Product Management?” lecture down your throat—I’ve learned the hard way that that just leads to eye rolls and abrupt ends in conversation.

Instead, I want to look back on some memorable career moments, clarify some of the lessons learned, and then use them to paint a picture of how they’ve impacted my personal life—particularly around discovering what gives me joy, purpose, and energy.

I’ll start with a funny story to show where I'm going with this.

A few years ago, I was a freshly minted new grad excited to attend his first corporate holiday party. I was *especially* excited about the open bar and pretty much hovered around that area with my friends for most of the evening.

When it came time to leave, I was part of this cabal of inebriated young professionals frantically trying to figure out what to do afterward. Picture a group of type-A, 22-year-old product managers trying to debate what constituted a proper night out in San Francisco.

I probably voiced some strong opinions loudly and obnoxiously (i.e. "Guys....Arena?!").

During all this craziness, I met my friend's date. This poor girl had already endured an evening full of high-energy, corp-speaking yuppies and was clearly fed up with having more overly-confident opinions forced upon her. She grabs me on the shoulder and screams: “CAN YOU **NOT** USE YOUR P.M. VOICE FOR JUST ONE SECOND?!”

I’m still wrestling with the ensuing embarrassment but I'll use this as a natural springboard for a deeper point I’m trying to make.

I make a conscious effort to separate my job from my identity—product management is "what I do" not "who I am". With that said, I’d be lying if I didn’t admit that what I do does influence who I am at least to some degree.

If you were wondering how I responded to this request to "turn off my PM voice", I shouted back with an equal amount of bluster “I’m sorry! I can’t!”

It was an odd moment of clarity where I was essentially acknowledging how it's impossible to fully compartmentalize my work and personal life.

They bleed into each other and today I want to explore how.

The PM job description is pretty nebulous—you're on the hook to make stuff happen by really any means necessary. But if I had to decompose it, I'd say my job is to ensure everyone knows what to do next, why they’re doing it, and how their work maps to the overall success of the team.

We'll get into the tactical details in a second but my favorite part of this job—hands down—is being around talented, motivated people.

Watching people operate at peak performance is inspiring and I love the process of figuring out how I can amplify their talent and impact while we all work towards a common goal. There's a responsibility to bring out the best in people similar to what I want to do for my personal relationships.

It's a People job. And to do it well, I work on the same sort of skills that I believe also make my friends feel energized, supported, and appreciated.

My day-to-day boils down to trying to do three basic things very very well:

1. Earning trust
2. Communicating ideas
3. Making good decisions

While I’m still early in my career, I’ve learned valuable lessons about all three and I want to reflect upon the ways they've impacted my personal life as well.

# Earning trust

![21-year-old Phil. Self-awareness clearly lacking](https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2Ffdf49618-1dc8-4453-a1c6-176c58640b61_715x1095.jpeg)

21-year-old Phil. Self-awareness clearly lacking

Picture a 21-year old Phil. I landed a PM internship at Google that summer and was still riding the high from all the external validation I was receiving from friends and family.

I was invited to panel Q&A sessions for people interested in the internship. I was reviewing resumes. I was fielding calls from random people my mom had told at church. And all this before I had even started my internship.

It's funny how a tiny bit of success can trick you into thinking you actually know what you're talking about.

Tasked with redesigning the mobile search experience for events and concerts, I thought I was going to be the Steve-f***ing-Jobs of this relatively under-invested corner of Google Search.

I was going to descend upon this team of engineers and designers—like Moses bringing down the Ten Commandments from Mount Sinai—showcasing my full arsenal of incredible strategic thinking, ingenious product insights, and extraordinary interpersonal skills while inspiring their loyalty and adoration in the process.

My ego was as inflated as its ever been in my lifetime.

Despite being an intern, I had a surprising amount of autonomy to lead this team and project. While we accomplished a few of the goals we set out that summer, we never achieved the sort of trust or even output I believe high-functioning teams demonstrated.

I envisioned launch parties and congratulatory handshakes from executives but instead received some painful lessons about relationship building that I still carry to this day.

My overly inflated ego had an unhealthy need to be validated. I didn't want my team to just like me, I wanted to be venerated like the iconic coaching personalities from some my favorite sports movies ("[Let's win one for the Gipper!](https://youtu.be/7yHPTdliKmk?t=14)").

I prepared for every meeting and presentation. I engaged in small talk and smiled politely when folks would tell me about their weekends. I did everything I could to come across as smart, capable, and charismatic—the boxes I felt needed checking in order to win over my team.

All this made some of the end-of-internship feedback all the more jarring. This was *actual* feedback I received from my engineering lead: *“**I honestly didn’t think you were that useful for most of the internship.**”*

Clearly, I wasn’t the elite product manager I thought I was.

Thankfully, I had the presence of mind to dig into that harsh criticism (after going through all the various stages of grief). It wasn't so much my quality of work or the insights I was providing, it was the team feeling like I would constantly index on my own ideas rather than leveraging some of the deep expertise and experience we had on our team.

"You need to be a better listener " was the most succinct and impactful piece of constructive criticism I received that summer.

And it was true. I was an absolutely terrible listener. Sure, I feigned interest—nodding my head, repeating what they said, and checking off all the boxes laid out in your typical manual about good “active listening”.

But I wasn’t really listening. I was just waiting for my turn to speak—cherrypicking my attention, hunting for a moment where I can say something intelligent.

Conversations with me were mostly transactional and served what I needed to get done.

My desire to be *perceived* as a leader was ultimately undermining my efficacy as one.

As luck would have it, all this crested around the same time that my then-girlfriend expressed not feeling particularly happy about our relationship. Shocker...she explained how she rarely felt like I was actually listening to what she had to say. Instead, I was either dismissing the problems she was facing (i.e. “I think you’re overreacting”) or trivializing them by offering knee-jerk solutions (i.e. “Oh you should just do this”).

While I wanted her to be happy, my behavior reflected someone who (selfishly) wanted whatever discomfort she was going through to just...go away. I didn't want to empathize with her problems, I just wanted them to evaporate.

She—like every sensible human being—just wanted to feel heard and valued.

Needless to say, this two-pronged assault on my personal and professional life forced some hard introspection. Fortunately, I had some great people in my life to model myself after.

I started thinking about people I most respected—the ones I would literally run through a wall for. I realized that all of them have this uncanny ability to make you feel like the only person in the room. **They inspire trust not by saying the right things but by being absolutely incredible listeners.**

They listen without listening for anything in particular. They listen openly as opposed to listening to defend their own ideas or for opportunities to offer their own opinions. They make their sole, all-encompassing purpose the other person and what they have to say.

They’re engaged, they’re curious, and they do everything in their power to provide you with this deep, universal need to feel understood and accepted. And even if they ultimately disagree with you, because they listened so well, you know that the decision came from a place of integrity and that your opinions were very very carefully considered.

That summer was a wake-up call and it started me on this journey of becoming a better listener both for my job and to improve my relationships with the people I love.

I’m nowhere near perfect, but I've seen dividends from investing in this new philosophy around  relationship-building.

I believe whole-heartedly that whenever someone decides to spend time with you, to patiently explain what’s going on, and to share what they’re feeling, they’re making a valuable investment in your relationship. And if we don’t treat that interaction with the level of respect it deserves, we’re sending a signal that we don’t value their ideas, their time, or their connection.

Relationships are built upon a foundation of trust. That foundation isn’t crafted with panache nor a desire to impress people, but with the consistent practice of making others feel like you're truly there for them.

# Communicating ideas

![https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2Fed48fbd0-0136-45af-9bac-2ca3a7dd0591_640x586.jpeg](https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2Fed48fbd0-0136-45af-9bac-2ca3a7dd0591_640x586.jpeg)

Fast forward two years later. I was a few weeks into my stint working on the Android Messages team and I walked out of my first presentation thinking I'd absolutely crushed the Q&A portion.

I had confident, detailed responses to every question—even ones I didn't know the answers to!

So you can imagine my surprise when my boss tapped me on the shoulder afterward and told me “Hey…you just said a lot of words without saying anything at all.”

Much like my journey to become a better listener, this journey also started with a gross overestimation of my capabilities.

My boss smiled at my disappointed expression and explained, “You're not just putting words together, you're giving people what they need. Work on that!”

Here's the lesson: **it’s not what you say, it’s what people hear**. And clearly, what I was saying wasn't resulting in people hearing anything particularly meaningful.

Conversely, what my boss communicated was loud and clear. I needed to do a better job translating the jumbled mess of ideas in my head into language that is actually useful for people around me.

Language is powerful precisely because of how much information we *don’t* actually need to convey something. Detail-rich ideas and experiences can be synthesized into just a few words which means every word we choose contains an awesome power.

For example, imagine that I told you “I burned my hand.” It's only four words but you can immediately fill in the details behind that message. You’re imagining my pain, you’re visualizing the incident, and—if you happen to follow me on Instagram—you probably inferred this happened because I was cooking this past weekend.

Now imagine that same event expressed in less precise language: “My hand hurts today.” Well…when did it start hurting? How did this happen? Why are you like this?

A playbook of effective communication tips isn’t something I’m super qualified to offer but I’ll highlight a few principles I’ve been actively thinking about.

## **The question behind the question**

Great communicators don't just word vomit every little piece of information. They curate what information is most valuable for the people they're talking to.

For example, when someone asks “How is the project going?” they’re probably not looking for your personal feelings about the actual work. More likely, they’re wondering if you’ll hit your deadline, they’re worried about some unknown risk, or they want to know if the decision to work on this project is still the right one. There's some unspoken anxiety and the message you communicate should quell that anxiety to some degree.

There's always a question behind the question. Effective communication doesn't just answer a question at face value, they help solve a deeper underlying need.

Don’t just think about what you want that person to *know* but rather what you want them to *feel.*

## **A clear and consistent blueprint**

Our communication acts like a bite-sized blueprint where others fill in the rest of the message with their own imagination.

The words we choose, the tone we adopt, and our non-verbal cues all contribute to blueprints that are then interpreted uniquely by each individual.

If I'm telling someone that I appreciate them, am I leaning in? Am I making eye contact? Am I smiling genuinely?

If my arms are crossed, I'm leaning back, and I'm looking out into the distance, it's unlikely they sense any gratitude at all no matter what words I use.

Since writing more frequently, I've been much more mindful of how all these things can impact the underlying message. Is there a simpler word that paints a more accurate picture? Is my tone consistent with the feelings I'm trying to convey? Do my non-verbal cues amplify or contradict my message?

Whether it's getting something done at work or strengthening our existing relationships, we need to be clear about our thoughts, feelings, and intentions.

And to do that, I've learned that communication is only meaningful when it's meaningful to others and not just to ourselves.

# Making good decisions

![I’ve never made a bad decision when it comes to fashion](https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2Faee5d793-4b8b-4fbc-8b3a-1d1089dcb2b2_604x453.jpeg)

I’ve never made a bad decision when it comes to fashion

This is kind of the point of it all am I right?

Building trust and communicating clearly gives us a shot at doing the right thing even when the right thing isn’t immediately obvious.

Should we work on this project or that project? Do we move forward with this idea or put it on hold? Do we launch now or wait until we have more information?

These are the sort of discussions I have with my team and the stakes are fairly high. Not so much because everyone is waiting on bated breath for the next Android Messages feature (although I like to think they do), but more so because these choices ultimately impact how people are going to spend their valuable time and energy in the weeks, months, and quarters ahead.

And if I really value my people and care about their work, it’s my responsibility to ensure that what we decide to do is both impactful and worthwhile even though it's not always clear which path we go down.

How do I work through risk and uncertainty? How do I counteract my blind spots and biases? How do I even track whether or not I’m becoming a better decision-maker?

Figuring out how to answer these questions not only helps me do my job better but they're also helpful in personal matters like figuring out where to eat or picking your partner.

There are entire books and business school courses dedicated to the art of decision-making. Like the previous sections, I won’t be offering anything novel, but I will pick up a few helpful tools that have served me well over the years.

## **Ask the right questions**

My job isn’t to *be* right, it’s to make sure we’re *doing* the right things.

Because I’m neither a psychic nor a human encyclopedia, I believe most of my energy should be spent asking the right questions rather than providing the right answers.

There’s really an art to it. A great question is an exercise in truth-seeking and also ensures everyone is using the same frame to think about the problem and decision.

Yes, some truths will be objective and agreed upon—like how many feet there are in a mile. But other truths will be subjective and non-obvious with the current information we have available—like how cool does Phil look with transition lenses

The one rule with truth-seeking is that you should be open to the truth described by the answering party. A question is no good if you’ve already decided the answer yourself.

Great questions also make sure people are on the same page. When starting a project, most people ask: “Will this succeed?”. But we can't answer that question without first answering: “What does it *mean* for this project to succeed?”

One of the really fun and challenging parts of my job is figuring out ways we can answer subjective questions like “How do we know this feature makes people feel more connected?”.  “How do we quantify how much our brand is ‘improving’?” Or my personal favorite: “When can we start patting ourselves on the back?”

The methodologies quantifying seemingly intangible things are beyond the scope of this post, but the takeaway here is that great questions establish a common understanding of what a good outcome looks like **before** making a decision.[1](https://www.notion.so/Can-you-NOT-use-your-PM-voice-92e7037ab66344c284bfc361d132305a)

## **Try to reduce uncertainty but don’t expect to eliminate it**

In a famous study about how people think about risk, researchers asked parents how much they would hypothetically pay for a treatment that would reduce their child’s risk of contracting a serious illness from 10% to 5%. They then asked how much they'd pay to reduce it from 5% to 0%. As you would expect, even though the reduction is the same magnitude for both cases, parents were willing to pay up to three times more to reduce the risk from 5% to 0%.

Human beings crave certainty and we’re willing to go to great lengths to achieve it even if it's impossible for most decisions we make.

Whether it’s having people vote, relying solely on the opinion of one respected person, or adding up a personal pros-and-cons list, there's comfort in employing a deterministic, discrete process when confronted with probabilistic, uncertain outcomes.

Too often, our instinct to eliminate uncertainty entirely makes us frame decisions as metaphorical forks in the road. The outcome is either good or bad based on a singular choice when in reality there’s actually a wide spectrum of possible outcomes based on a broader set of factors.

We tend to ask: “Should we do X?” instead of “What needs to be true for X to be the right choice?”

“What needs to be true?” is actually my favorite example of a great question. It accepts the fact that what we’re really doing is making the best possible, educated guess on what to do with the information we have.

The question audits what we know with certainty but also forces us to consider what must be real and what must be universally accepted for us to move forward down a particular path. A binary choice is decomposed into various avenues with which we can gather more information to make a better decision.

I wrote a little bit about one-way vs. two-way door decisions in a [previous post](https://bookie.substack.com/p/my-favorite-ideas-of-2021?s=w). A surprising amount of decisions are two-way doors—you walk through in one direction and if things don't work, you can always double back.

Deciding is usually the hardest part and we’re better off being fast and wrong than slow and right. While you tread cautiously and patiently through one-way doors, most decisions (much like most things in life actually) aren't worth losing sleepover.

I try to remind my friends of that when it takes an entire day to decide where we're going to eat.

So when in doubt, pick the path that lets you efficiently gather the most signal. Decide, learn, iterate, and then take that next critical step that puts you closer to your goal.

## **Don’t tie your identity to your decisions**

My early career blunders taught me a pretty valuable lesson: you don’t need to be the smartest person in the room to add value.

It’s a trap all knowledge workers face: we treat our ideas like personal property that needs to be protected and defended. After all, how are we valuable if not through the ideas we bring to the table?

But I've seen first-hand how destructive—and frankly exhausting—it can be to feel like you always need to be right.

Instead, detach your self-worth from your beliefs. Don’t try to be right all the time, try to figure out what’s true. Be open to changing your opinions when new information arrives. As the saying goes: “Strong opinions, loosely held”

That’s not to say that we shouldn’t have *any* opinions--we all have points of view and valuable insights to offer. Conviction is important but we should be careful that our conviction doesn’t turn into commitment.

Commitment can lead to other behavioral mistakes like perceiving only segments of reality that conform to our beliefs (i.e. confirmation bias) and then reinforcing those beliefs with echo chambers of like-minded people.

I’m not smart enough to know with absolute certainty whether something is going to happen and neither are you.

The best people I've worked with consistently showcase fair, principled, and rigorous decision-making.

They “show their work”— communicating the critical steps in their thought process, the facts they know, and the assumptions they hold while also taking in information from those around them.

They don’t strive to always be right because they know it’s statically impossible. Instead, they focus their energy on the values I highlighted previously.

Do people feel heard and their ideas valued? Are we asking the right questions? How can we learn as much and as quickly as possible?

Much like how I want our team to be doing the right things, I also want to be someone my friends can depend on. To do both, people need to trust that my desire to be thoughtful, fair, and intellectually honest is greater than my desire to be right.

# Conclusion

If you had asked me why I wanted to be a product manager early in my career, I would’ve parroted something like “I want to build things people use” or “I want to exercise more strategic thinking.” Also if I’m being perfectly honest, there was definitely some aspect of having “manager” in my job description that was also really appealing.

And while a number of those reasons still hold true, what's surprised me most has been how fulfilling the people aspect of this job has been. I cherish congratulations and backslaps as much as anyone who gets to take credit for great work, but really what puts a big smile on my face after a long day of meetings, emails, and writing documents is feeling like I made someone’s day just a little bit better.

I provided a new piece of context, offered a new way to look at something, or supplied some much-needed gratitude so that talented people can do their best work.

Most people I encounter seem to have very clear goals and destinations in mind for their careers—either a specific job title or a set of achievements. I have some of that too but I've noticed that I tend to focus more on helping and less on achieving.

More on building and less on winning. More on honing the skills we talked about rather than the results that come with applying them.

Needless to say, I don’t always succeed. Some days I ramble, make poor decisions, and hurt more than I help. But I love that process of showing up every day and continuing to practice these values that have gradually diffused to all corners of my professional and personal life.

I want my friends and family to feel heard and valued. I want to express my thoughts and feelings in ways that strengthen our relationship. I want them to trust my decisions and be open to me guiding them through their decision-making as well.

Maybe these qualities just come naturally with growing up, but I can say unequivocally that my job has helped immensely in that process .

So do I think I'll be doing this job for the rest of my life?

Who knows.

But I'm excited to see where these personal values take me in the years ahead!

---

## Additional Content

I mentioned a few of these takeaways in my 2021 [Year-in-Review](https://bookie.substack.com/p/my-favorite-ideas-of-2021?s=w) but here are some of my favorite reads about listening, communication, and decision-making:

**Listening and Communication**

- 📚 [On Writing](https://www.amazon.com/Writing-Memoir-Craft-Stephen-King-ebook/dp/B000FC0SIM/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1649773606&sr=1-1) by Stephen King
- 📚 [Doesn’t Hurt to Ask](https://www.amazon.com/Doesnt-Hurt-Ask-Questions-Communicate-ebook/dp/B084GBT41F/ref=sr_1_1?crid=2FDG5EUEFG5K8&keywords=doesn%27t+hurt+to+ask&qid=1649773726&s=digital-text&sprefix=doesn%27t+hurt+to+ask%2Cdigital-text%2C159&sr=1-1) by Trey Gowdy
- 📚 [Building A Story Brand](https://www.amazon.com/Building-StoryBrand-Clarify-Message-Customers/dp/0718033329/ref=sr_1_1?crid=21UKE80SA94TV&keywords=building+a+storybrand&qid=1649773759&s=books&sprefix=building+a+storybra%2Cstripbooks%2C84&sr=1-1) by Donald Miller
- 📚 [Stop Talking, Start Communicating](https://www.amazon.com/s?k=Stop+Talking%2C+Start+Communicating&i=digital-text&crid=2AFB33HMI614M&sprefix=stop+talking%2C+start+communicating%2Cdigital-text%2C173&ref=nb_sb_noss_2) by Geoffrey Tumlin

**Decision-Making**

- 📚 [Superforecasting](https://www.amazon.com/Superforecasting-Science-Prediction-Philip-Tetlock/dp/0804136718) by Philip Tetlock.
- 📚 [How Not to Be Wrong: The Power of Mathematical Thinking](https://www.amazon.com/How-Not-Be-Wrong-Mathematical/dp/0143127535/ref=sr_1_1?gclid=Cj0KCQjwxtSSBhDYARIsAEn0thSf8KUDJBPFF3UbxsO2Ih3QB6F6u_CfZ7WLPeLI4nSVi_McAek_PJkaAhduEALw_wcB&hvadid=409909962621&hvdev=c&hvlocphy=1022762&hvnetw=g&hvqmt=e&hvrand=14933438602828333122&hvtargid=kwd-86825105916&hydadcr=22538_11318434&keywords=how+to+not+be+wrong&qid=1649773575&s=books&sr=1-1) by Jordan Ellenber
- 📚 [Think Like a Rocket Scientist](http://xn--%20think%20like%20a%20rocket%20scientist%20by%20ozan%20varol-um294d/) by Ozan Varol
